### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by romualdszkudlarek**
- created a Document-fr directory
- copied all files from parent directory ("Document") in English to the Document-fr directory (hence all files were in English originally)
- translated 0x01/2/3

Did not modify any file other than in Document-fr.

Thanks for merging the new directory with the Master branch content so that the effort becomes visible and others can contribute :-) (in particular, proofreading is welcome on translated sections)

### CODE REVIEW

File: Document-fr/0x02-Frontispiece.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Frontispice
+
+## A propos Du Standard
```
Comment (commjoen) [inline Document-fr/0x02-Frontispiece.md]: `A Propos du Standard` (capitalization)

File: Document-fr/0x02-Frontispiece.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+http://owasp.herokuapp.com/.
+
+## Copyright et Licence
+
+![license](images/license.png)
+Copyright © 2018 The OWASP Foundation. Ce document est publié sous la licence Creative Commons Attribution ShareAlike 3.0. Pour toute ré-utilisation ou distribution, il est obligatoire d'attribuer la licence de ce travail aux auteurs.
+
+| Chef de Projet | Auteur Principal | Contributeurs et Relecteurs
+| --- | --- | --- |
+| Sven Schleier & Jeroen Willemsen | Bernhard Mueller | Stephen Corbiaux, Sven Schleier, Anant Shrivastava, Abdessamad Temmar, Alexander Antukh, Roberto Martelloni, Stefaan Seys, Prabhant Singh, Francesco Stillavato, Abhinav Sejpal |
```
[the remark this answers is not in the document]
Comment (romualdszkudlarek) [reply Document-fr/0x02-Frontispiece.md]: Done

File: Document-fr/0x02-Frontispiece.md (hunk 1 above)
Comment (romualdszkudlarek) [reply Document-fr/0x02-Frontispiece.md]: Done

**Comment by romualdszkudlarek** [comment]
Changes implemented

File: Document-fr/0x02-Frontispiece.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Frontispice
+
+## A Propos Du Standard
```
Comment (commjoen) [inline Document-fr/0x02-Frontispiece.md]: Du --> du

**Comment by romualdszkudlarek** [comment]
Done

**Pull request merged**